cli/args/flags.rs

```diff
@@ -1875,6 +1884,13 @@ Directory arguments are expanded to all contained files matching the glob
         .require_equals(true)
         .default_missing_value("-")
     )
+    .arg(
+      Arg::new("dot-reporter")
```

Comment:
Should it be `--reporter=dot`?

Reply:
I was thinking that as well, but with `--reporter=junit` how are we gonna specify the path? Add a separate `--junit-path` flag?

That is actually very valid in context of adding this configuration to `deno.json`.

Reply:
I think you want junit output _in parallel_ to the visual CLI reporter. So I think in that sense, it's not a replacement for the main reporter, but something additional. This would suggest that `deno test --reporter=dot --junit-path=./junit.xml` should be valid.

Reply:
That's a reasonable solution. I'll update the PR.

Reply:
+1 on supporting multiple reporters at the same time.

Reply:
While pretty+junit and dot+junit make sense, pretty+dot doesn't. So I'm gonna err on what Luca suggested here.